script/glide.sh

```diff
@@ -49,7 +36,7 @@ if ! type glide-vc > /dev/null 2>&1; then
   exit 1
 fi
 
-if [[ ! -e "${CWD}/glide.yaml" ]]; then
+if [[ ! -e "glide.yaml" ]]; then
```

Comment:
For the sake of simplicity, I would prefer to use this terrible trick only when needed ^^